frontend/src/pages/AnnouncementPage.js

```diff
@@ -10,21 +10,27 @@ import useChangeRoute from "../hooks/useChangeRoute";
 import { Link as RouterLink, useLocation } from "react-router-dom";
 
 import Grid from "@mui/material/Grid";
-import {
-	Button, Alert,
-	TextField, Typography,
-} from "@mui/material";
+import { Button, Alert, TextField, Typography } from "@mui/material";
 import Box from "@mui/material/Box";
 import Collapse from "@mui/material/Collapse";
-// import Link from "@mui/material/Link";
+import Link from "@mui/material/Link";
 
 import RoundedEdgesButton from "../components/RoundedEdgesButton";
 import HeaderTypography from "../components/HeaderTypography";
+import ParagraphTypography from "../components/ParagraphTypography";
+
+import examplePhoto1 from "../assets/img/announcementPhotoExamples/anouncement_1.png";
+import examplePhoto2 from "../assets/img/announcementPhotoExamples/anouncement_2.png";
+import examplePhoto3 from "../assets/img/announcementPhotoExamples/anouncement_3.png";
+import examplePhoto4 from "../assets/img/announcementPhotoExamples/anouncement_4.png";
+import examplePhoto5 from "../assets/img/announcementPhotoExamples/anouncement_5.png";
+
+const photos = [examplePhoto1, examplePhoto2, examplePhoto3, examplePhoto4, examplePhoto5];
 
 function InputFields({ defaultTitle, defaultContent, titleComponenId, contentComponenId }) {
 	// TODO: formated text support
 	// ref: https://mui.com/components/text-fields/#integration-with-3rd-party-input-libraries
-	return <Grid sx={{ mt: 3, mb: 3 }}>
+	return <Grid sx={{ mt: 3, marginBottom: 3 }}>
```

Comment:
I know, this change is actually unnecessary, but I forgot to undo it. I hope, this is okay.